### PR TITLE
Documentation for Publishing UDFs as Remote Packages

### PR DESCRIPTION
**Pull request by nandorsoma**

### CODE REVIEW

File: docs/reference/sqrl/functions/custom-functions.md [hunk 1]
```diff
@@ -14,3 +14,50 @@ Then follow the instructions below for implementing a function package in the la
...
+{
+  "version": "1",
+  "package": {
+    "name": "datasqrl.examples.udf",
+    "version": "0.5.5",
+    "latest": true
+  }
+}
+```
+
```
Comment (mbroecheler) [inline docs/reference/sqrl/functions/custom-functions.md:38]: Can you add a link here to the main documentation page for the package.json: "For more details on how to configure the package manifest, ..."

File: docs/reference/sqrl/functions/custom-functions.md [hunk 2]
```diff
@@ -14,3 +14,50 @@ Then follow the instructions below for implementing a function package in the la
...
+## Upload User-Defined Functions to the Repository
+
+If you want to upload your custom UDF to the Datasqrl repository, you need to take some extra steps.
+
+In the same folder where the JAR file resides (e.g., `myudf-0.1.0-SNAPSHOT.jar`), create the following files.
+
+### Create `package.json`
+
+The `package.json` file defines the package metadata, such as name, version, etc. A minimal `package.json` looks like this:
```
Comment (mbroecheler) [inline docs/reference/sqrl/functions/custom-functions.md]: We use the word "manifest" instead of "metadata" since the latter is overloaded.

**Review by mbroecheler — APPROVED** [review]
Some minor changes. Otherwise looks good!

**Comment by nandorsoma** [comment]
Thank you @mbroecheler. I updated the doc based on your suggestions.

**Pull request merged**